lib/Convos/Core/Backend/File.pm

```diff
@@ -80,13 +80,13 @@ sub messages {
   $re = qr{\Q$re\E}i unless ref $re;
   $re = qr/^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}) (.*$re.*)$/;
 
-  $args{after}  = Time::Piece->strptime($query->{after},  '%Y-%m-%dT%H:%M:%S')if $query->{after};
-  $args{before} = Time::Piece->strptime($query->{before}, '%Y-%m-%dT%H:%M:%S') if $query->{before};
-  # If neither before nor after are defined, search everything before now
-  $args{before} = gmtime if !$args{before} && !$args{after};
   $args{limit}    = $query->{limit} || 60;
   $args{messages} = [];
   $args{re}       = $re;
+  # TODO: What to do when after is after before?
```

Comment:
"before" should be a later date, i.e. we are searching for messages "before" a given date.
"after" should be an earlier date, i.e. we are searching for messages "after" a given date.

As such, after should come before, for a normal "between".  But what happens if we are supplied a query where "after" comes after "before"?